hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSPermission.java

```diff
@@ -260,6 +261,16 @@ private void createAndCheckPermission(OpType op, Path name, short umask,
     checkPermission(name, expectedPermission, delete);
   }
 
+  @Test
+  public void testFSNamesystemCheckAccess() throws Exception{
+    Path testInvalidPath = new Path("/test2");
+    fs = FileSystem.get(conf);
+
+    LambdaTestUtils.intercept(FileNotFoundException.class,
+            "Path not found: " + testInvalidPath,
+            () -> fs.access(testInvalidPath, FsAction.READ));
```

Comment:
There is a new checkstyle warning about the alignment of this statement.

Reply:
Thanks @virajjasani  for you review, have fixed the complain